smithy-cli/src/test/resources/software/amazon/smithy/cli/commands/smithy-lock-test.json

```diff
@@ -0,0 +1,11 @@
+{
+  "_AUTOGENERATED_FILE_DO_NOT_MODIFY_MANUALLY_":"",
+  "version":"1.0",
+  "config_hash":-1856284556,
+  "repositories":["https://repo.maven.apache.org/maven2"],
+  "artifacts":{
```

Comment:
Would this lockfile ever contain other dependency types? If so, we might want to put this in a maven section.

Reply:
I suspect we may want to do that someday however, my thinking was that we would include the type as a property of the artifact rather than as a section. I'd rather hold off on this for now and do a new version of the lockfile once we have users asking for additional dependency types.